lib/src/similar_web_service_base.dart

```diff
@@ -1,22 +1,56 @@
+import 'package:dio/dio.dart';
+import 'package:similar_web/src/api_constants/api_constants.dart';
+
 /// {template ApiSimilarWebServiceBase}
 /// ApiSimilarWebServiceBase is the base class for all API Request available.
 /// {endtemplate}
 abstract class ApiSimilarWebServiceBase {
```

Comment:
yo iría con un approach de una sola clase (no abstract) con extensions x cada gilada q necesitemos y fue... no em gastaría mucho en parafernalias q al final no van a ser de ayuda y con un simple ctrl + . ya tenemos todo loq puede darnos y en su defecto una maenra muy sencilla de _extender_ el paquete para quien necesite algo mas... bien facil
_simplificity is the ultimate sophistication_

Reply:
Hechoo, tendrias que hacer un doble check, para ver que este clean